### ISSUE TITLE
Update OpenInSandboxAction action text overrides for 2024.1

### ISSUE THREAD

**Issue by stachenov** [apollographql/apollo-kotlin#5446]
### Description

Hello,
Starting from 2024.1 the IJ platform will use a new place ID for "Reveal In" / "Open In" actions, namely `RevealInPopup`. This is necessary because popup menus can be customized, and the action name should depend on where exactly in the popup menu the action is located. That is, when located under "Open In", then the name "X" would be read as "Open In X", but if the same action is moved to the top level, then the "X" alone is confusing. See [IDEA-339585](https://youtrack.jetbrains.com/issue/IDEA-339585) for details.

In order for this new mechanism to work, this
```
      <override-text place="EditorTabPopup" />
      <override-text place="ProjectViewPopup" use-text-of-place="EditorTabPopup" />
      <override-text place="EditorPopup" use-text-of-place="EditorTabPopup" />
      <override-text place="FavoritesPopup" use-text-of-place="EditorTabPopup" />
```
should be replaced by this:
```
      <override-text place="RevealInPopup"/>
```
It automatically covers all popups that contain the `RevealGroup` action group.

**Comment by BoD** [comment]
Thanks a lot for reaching out! 👍 I'll update the manifest.

**Comment by stachenov** [comment]
Note that this new place ID isn't available yet in 2023.3.*, so it only should be updated for 2024.1 when that's out.
